gtk-3.0/gtk-widgets.css

```diff
@@ -2052,8 +2054,8 @@ scrollbar.overlay-indicator trough,
 
 scrollbar:hover trough,
 .scrollbar:hover.trough {
-    background-color: alpha (#000, 0.1);
-    border-radius: 20px;
```

Comment:
Maybe trying moving these styles up to `scrollbar.overlay-indicator trough` so we can preserve them for the overlay version?